llvm/tools/file-table-tform/file-table-tform.cpp

```diff
@@ -217,6 +245,16 @@ struct TformCmd {
                         Table.replaceColumn(Args[0], *Table1->get(), Args[1]);
                     return Res ? std::move(Res) : std::move(Error::success());
                   })
+            .Case(OPT_REPLACE_CELL,
+                  [&](TformCmd *Cmd) -> Error {
+                    assert(Args.size() == 2 && Cmd->Inputs.size() == 1);
+                    const int Row = std::stoi(Args[1].str());
+                    if (Row > Table.getNumRows())
```

Comment:
This is checked twice here and in updateCellValue method.

Reply:
I've remove this check and added the argument comment similar to the other cases.